extensions/kafka-connect/src/main/java/com/hazelcast/jet/kafka/connect/impl/ReadKafkaConnectP.java

```diff
@@ -132,7 +132,7 @@ private BroadcastKey<String> snapshotKey() {
     protected void restoreFromSnapshot(@Nonnull Object key, @Nonnull Object value) {
         boolean forThisProcessor = snapshotKey().equals(key);
```

Comment:
if we use a shared state object maybe we should use the same snapshot key for each processor? Any thoughts?

Reply:
Actually that's a good question. Can you check if snapshots are saved with the same values?